packages/webcomponents/src/api/services/apple-pay.service.ts

```diff
@@ -16,6 +16,15 @@ import {
   IApplePayToken,
 } from '../ApplePay';
 
+// Centralized error codes for Apple Pay service
+export enum ApplePayServiceErrorCode {
+  APPLE_PAY_UNAVAILABLE = 'APPLE_PAY_UNAVAILABLE',
```

Comment:
When this is prefixed, won't it be `APPLE_PAY_APPLE_PAY_UNAVAILABLE`? Should we just write them all out here and not prefix later to make things simple? Otherwise maybe just change this to `UNAVAILABLE`?